packages/mml-web/src/elements/Audio.ts

```diff
@@ -100,6 +145,18 @@ export class Audio extends TransformableElement {
         parseFloatAttribute(newValue, defaultAudioVolume),
       );
     },
+    "cone-angle": (instance, newValue) => {
+      instance.audioAnimatedAttributeHelper.elementSetAttribute(
+        "cone-angle",
+        parseFloatAttribute(newValue, null),
```

Comment:
I think it makes sense to clamp these values (0 - 360) before they go into the animated attribute helper